### PR TITLE
[PL] [00000] version bump for few gems

### PR DESCRIPTION
**Pull request by pasha**
- mediawiki
- rake
- rspec
- ruby 2.1.3
- version bump 2.2.2

### CODE REVIEW

File: .ruby-version.example [hunk 1]
```diff
@@ -1 +1 @@
-1.9.3-p327
+2.1.3
```
Comment (c0) [inline .ruby-version.example]: What do you think about dropping `.example` from this file and going with `.ruby-version`?

Comment (pasha) [inline .ruby-version.example]: i like it, will change

**Comment by c0** [comment]
:+1: 
I also added `bin` to `.gitignore`

**Comment by pasha** [comment]
thanks!

**Pull request merged**